examples/react/full-width-resizable-table/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "tanstack-table-example-full-width-resizable-table",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "serve": "vite preview",
+    "start": "vite"
+  },
+  "dependencies": {
+    "@faker-js/faker": "6.3.1",
+    "@tanstack/react-table": "8.0.0",
+    "faker": "^6.6.6",
```

Comment:
you don't need faker dependency as you already have faker-js.

Reply:
faker removed from deps